### ISSUE TITLE
v1.0.3.0 not working in VR (FS24 Steam)

### ISSUE THREAD

**Issue by Pogman** [kayjay1c6b/MSFS2024_AutoFPS-by-kayJay_1c6#14]
Using the current 1.0.3.0 version with MSFS24 Steam version the TLOD / OLOD adjustments seem to be targeting the non vr mode. When in vr the TLOD + OLOD change but this is not reflected in the sim and as shown in the vr graphic settings.

In the screenshot autofps shows tlod 31 while the sim is 400.

![Screenshot 2024-12-28 045324](https://github.com/user-attachments/assets/dc2e54e2-c50c-4051-be4a-f4f5e29af23a)

**Comment by ospaju** [comment]
Hi, same problem here.
I used your app, I think it works best with virtual reality.
Please I would be very grateful if you could solve this problem.
Thank you very much for your great work and happy new year

**Comment by ospaju** [comment]
The solution is to delete it from the community folder:                                                                                                                            "mobiflight-event-module"

**Comment by Pogman** [comment]
> The solution is to delete it from the community folder: "mobiflight-event-module"

This didn't work for me.

The app installs this itself and removing it causes no connection to the sim session and thus no tlod/olod control.

**Comment by Pogman** [comment]
Yeah that's what I tried, deleted and reinstalled but the same issue remains unfortunately.

**Comment by ospaju** [comment]
Sorry, that worked for me. I also use VR and I find it to be an essential application. That's why I really appreciate the work done here, and any help is appreciated. Regards

**Comment by kayjay1c6b** [comment]
Hi, Could you tell me which version of the sim do you use? Steam or Ms
store? Do you also have installed autoFPS by Reset? I don't have any info
that  the app isn't working besides you.

śr., 8 sty 2025, 22:31 użytkownik ospaju ***@***.***> napisał:

**Comment by Pogman** [comment]
Steam version. This is the only autoFPS version installed.

Also using Pimax Crystal here. About the only thing I can tell is that being in VR mode is not detected by autoFPS and thus it's using and adjusting the flat settings which of course have no effect while in VR.

**Comment by kayjay1c6b** [comment]
This is something. I will try to look into it but have a newborn at home so
I don't know when I will have time.

śr., 8 sty 2025, 23:40 użytkownik Pogman ***@***.***> napisał:
